docs/service-and-repository-structure/service-and-repository-structure.md

```diff
@@ -0,0 +1,137 @@
+`NineChronicles.Headless` is mostly written in C#. You can build and run your local headless server
+following [Getting Started](https://github.com/planetarium/NineChronicles.Headless/wiki/Getting-Started).
+You can create and run your own blockchain network/node with this repository. Please
+check [Create new genesis block](https://github.com/planetarium/NineChronicles.Headless/wiki/Create-new-genesis-block)
+to create your own genesis block for your own network.
+
+### 3.2.1. Remark
+#### 3.2.1.1. Options for headless node
+There are several functions can be run using this repository.
+
+- Peer: save & spread available node addresses.
+- RPC server: Receives action data from client and make transaction to spread to peers.
+- GraphQL server: Returns data from blockchain to answer incoming GQL query.
+- Miner: The actual block miner. Collects transactions from peer nodes and verify tx.s to mine new block.
+
+You can run one or more functionalities at the same time in one node.
+See [README](https://github.com/planetarium/NineChronicles.Headless/blob/main/README.md) for all available options.
+
+#### 3.2.1.2. Usage of MagicOnion
+To make gRPC connection between Unity client and .NET server, we chose [MagicOnion](https://cysharp.github.io/MagicOnion/) as an implementation.  
+This is well-made server to meet our requirements, and distributed under [MIT license](https://opensource.org/licenses/MIT).  
+Although we're not using MagicOnion's feature very well inside logic, but this information we're using MagicOnion is useful for first-time viewer.
+
+## 3.3. NineChronicles
+
+#### [GitHub repository](https://github.com/planetarium/NineChronicles)
+
+The unity client to play NineChronicles Game. You need `Unity` to build and run this game on your local computer.  
+Following [Getting Started](https://github.com/planetarium/NineChronicles/wiki/Get-Started), you can build and run your own NineChronicles client on your environment.  
+If you want to enhance UX/UI or any user interactive properties, this is the right repository to dig.
+
+### 3.3.1. Remark
+
+#### 3.3.1.1. Unity version issue
+
+Due to prevent unintended issue, we highly recommend to use Unity version as we wrote in the document.  
+If you use later version instead noted version, you can meet a bunch of errors, and we cannot help at that situation.
+
+#### 3.3.1.2. Select blockchain to attach
+
+If you run your game using `Play` button on your Unity editor, It will run with its own internal blockchain network, not any existing external blockchain.  
+To attach your client to mainnet or other network, you have to change client configurations.
+
+## 3.4. Lib9c
+
+#### [GitHub repository](https://github.com/planetarium/lib9c)
+
+This is common library used broadly among NineChronicles game server and client.  
+This is [`Libplanet`](#3.5.-Libplanet) implementation to run NineChronicles game on libplanet blockchain network.  
```

Comment:
Isn't `Lib9c` `the protocol of Nine Chronicles game`?  I think this line needs others' opinions also.

My questions:
- `Libplanet` can be treated as blockchain network?
- `Libplanet` can be treated as an interface or a framework?

> Lib9c is a library that contains key implementations of Nine Chronicles, a decentralized RPG developed with Libplanet. Lib9c includes Nine Chronicle's key features like in-game decisions and data models, which can be used to implement game core capabilities. 

But I don't have a strong opinion about this so you can ignore this 🤔